Detections/ASimAuthentication/imAuthSigninsMultipleCountries.yaml

```diff
@@ -23,20 +23,24 @@ query: |
   let threshold = 2;
   imAuthentication
   | where TimeGenerated > timeframe
-  | where EventType=='Logon' and EventResult=='Success'
+  | where EventType == 'Logon' and EventResult == 'Success'
   | where isnotempty(SrcGeoCountry)
-  | summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), Vendors=make_set(EventVendor), Products=make_set(EventProduct)
-    , NumOfCountries = dcount(SrcGeoCountry)
-    by TargetUserId, TargetUsername, TargetUserType
+  | summarize
+      StartTime        = min(TimeGenerated)
+      , EndTime        = max(TimeGenerated)
+      , Vendors        = make_set(EventVendor)
```

Comment:
All make_set should have maximum allowed value.

Reply:
Added a max value.